include/ITS.Propagation.LFMF/LFMF.h

```diff
@@ -0,0 +1,109 @@
+/** @file LFMF.h
+ * Interface header for this library
+ */
+#ifndef __ITS_PROPAGATION_LFMF_LFMF_H__
+#define __ITS_PROPAGATION_LFMF_LFMF_H__
+
+#include <complex>
+#include <cfloat>
+
+#include "LFMFConfig.h"
+#include "ReturnCodes.h"
+
+using std::complex;
+using std::abs;
+
+namespace ITS {
+namespace Propagation {
+namespace LFMF {
+
+// Define cross-platform DLLEXPORT
+// Export the DLL functions as "C" and not C++
+#ifndef DOXYGEN_SHOULD_SKIP
+    #ifdef _WIN32
+        #define DLLEXPORT extern "C" __declspec(dllexport)
+    #else
+        #define DLLEXPORT extern "C"
+    #endif
+#endif
+
+#define MAX(x, y) (((x) > (y)) ? (x) : (y))                 ///< A macro that returns the maximum of \a a and \a b.
+#define MIN(x, y) (((x) < (y)) ? (x) : (y))                 ///< A macro that returns the minimum of \a a and \a b.
+#define DIM(x, y) (((x) > (y)) ? (x - y) : (0))             ///< A macro that returns the difference of \a a and \a b.
+
+#define PI                          3.1415926535897932384   ///< Approximate value of @f$ \pi @f$
+#define epsilon_0                   8.854187817e-12         ///< Vacuum permittivity (F/m) 
+#define a_0__km                     6370                    ///< Earth radius, in km
+#define C                           299792458.0             ///< Speed of light (m/s)
+#define SQRTPI                      sqrt(PI)                ///< Approximate value of the square root of @f$ \pi @f$
+#define THIRD                       1.0 / 3.0               ///< Value of one third 
+#define FALSE                       0                       ///< False
+#define TRUE                        1                       ///< True
+#define D2R                         PI/180.0                ///< Approximate convert ratio from Degrees to Radians
+#define R2D                         180.0/PI                ///< Approximate convert ratio from Radians to Degrees
+#define ETA                         119.9169832*PI          ///< Intrinsic impedance of free space (ohms)
+
+#define POLARIZATION__HORIZONTAL    0                       ///< Polarization horizontal
+#define POLARIZATION__VERTICAL      1                       ///< Polarization vertical
+
+#define METHOD__FLAT_EARTH_CURVE    0                       ///< Flat earth curve method
```

Comment:
Please define an enum `LFMF::SolutionMethod` to handle the solution method values